### PR TITLE
Add navigation bar, Add about me info, add resume

### PR DESCRIPTION
**Pull request by Jordan23-lab**

### CODE REVIEW

File: portfolio/src/main/webapp/about.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+      <header>
+          <nav>
+               <ul>
+    <li><a href="index.html">Home</a></li>
+    <li><a href="about.html">About Me</a></li>
+    <li><a href="resume.html">Resume</a></li>
+    <li><a href="activities.html">Activites</a></li>
+    <li><a href="accademic.html">Academic Achievements</a></li>
+    <li><a href="contact.html">Contact Us</a></li>
```
Comment (nitinms-sps-program) [inline portfolio/src/main/webapp/about.html]: Indentation: 2 spaces ahead of <ul></ul> level.

**Review by nitinms-sps-program — COMMENTED** [review]
Can you please send me a link to the profile page so I can see the html in action?

File: portfolio/src/main/webapp/about.html (hunk 1 above)
Comment (Jordan23-lab) [reply portfolio/src/main/webapp/about.html]: Yes here's the link to the website.

Website: http://jdrummond-sps-summer21.appspot.com/index.html

Comment (nitinms-sps-program) [reply portfolio/src/main/webapp/about.html]: Can you please indent the <li> to be 2 spaces ahead of <ul>...

e.g.:
```
<header>
  <nav>
    <ul>
      <li><a href=...
      <li><a href=...
    </ul>
```

**Pull request closed without merging**